README.md

```diff
@@ -1,2 +1,124 @@
-# School_Library
-The Ruby implementation of school library.
+
+<a name="readme-top"></a>
+<div align="center">
+<img src="https://cdn.icon-icons.com/icons2/2415/PNG/512/ruby_plain_logo_icon_146361.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>School Library</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Vet clinic database ](#-vet-clinic-database-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Create database](#create-database)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖School Library <a name="about-project"></a>
+
+
+**School Library** is a ruby project used to manage books and rent books to teachers and students.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ - Ruby
+
+### Key Features <a name="key-features"></a>
+
+- **Create Student and Teachers' credentials.**
+- **Create Rented Books.**
+- **Rent books to teachers and students alike.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby Runetime](https://www.postgresql.org/download/) installed
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```
+git clonehttps://github.com/calebchris000/School_Library.git && cd School_Library
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Caleb Nwaizu**
+
+- GitHub: [@githubhandle](https://github.com/calebchris000)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  **Add a GUI**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Mutai-Gilbert/Vet-Clinic-Database/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project please give it a ⭐ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank the Microverse Team! 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] Good job using the readme template provided by Microverse, however, the link to your MIT license is not working as expected as you can see in the screenshot below. please make sure that this works as well.
![afbeelding](https://github.com/calebchris000/School_Library/assets/39240820/ab7c79be-e584-4a05-a422-c299dff2a0b2)